src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int LOTTERY_START_NUMBER = 3;
+    private static Lottery lottery = new Lottery();
```

Comment:
In this case, I suggest not putting the lottery on a class level, the main method will be called only 1 time, so it is better to put it there